### ISSUE TITLE
VxCentralScan: remove unnecessary button

### ISSUE THREAD

**Issue by arsalansufi** [votingworks/vxsuite#4706]
We’ve got an unnecessary second click in a modal when saving CVRs and when a USB is already recognized in centralscan

Issue automatically created from Slack feedback reported by roe: 
https://votingworks.slack.com/archives/CJU9MSC6S/p1701471104407709

**Comment by adghayes** [comment]
I don't think this matters, and it's nice to reuse the component as-is from VxScan.
